components/usage/pkg/stripe/stripe.go

```diff
@@ -320,6 +320,55 @@ func (c *Client) GetSubscriptionWithCustomer(ctx context.Context, subscriptionID
 	return subscription, nil
 }
 
+func (c *Client) CreateSubscription(ctx context.Context, customerID string, priceID string) (*stripe.Subscription, error) {
+	if customerID == "" {
+		return nil, fmt.Errorf("no customerID specified")
+	}
+	if priceID == "" {
+		return nil, fmt.Errorf("no priceID specified")
+	}
+
+	params := &stripe.SubscriptionParams{
+		Customer: &customerID,
+		Items: []*stripe.SubscriptionItemsParams{
+			{
+				Price: &priceID,
+			},
+		},
+	}
+
+	subscription, error := c.sc.Subscriptions.New(params)
+	if error != nil {
+		return nil, fmt.Errorf("failed to get subscription with customer ID %s", customerID)
+	}
+
+	return subscription, error
+}
+
+func (c *Client) SetDefaultPaymentForCustomer(ctx context.Context, customerID string, setupIntentId string) (*stripe.Customer, error) {
+	if customerID == "" {
+		return nil, fmt.Errorf("no customerID specified")
+	}
+
+	if setupIntentId == "" {
+		return nil, fmt.Errorf("no setupIntentID specified")
+	}
+
+	setupIntent, err := c.sc.SetupIntents.Get(setupIntentId, nil)
```

Comment:
The secondargument here should specify at least the `ctx`. See other methods in this client.